lib/sierra-models/bib.js

```diff
@@ -23,6 +25,51 @@ class SierraBib extends SierraBase {
   isInRecap () {
     return this.items() && this.items().some((item) => item.location && item.location.code && item.location.code.startsWith('rc'))
   }
+
+  isOtfRecord () {
+    // Location code 'os' is a special location for temporary items
+    // https://github.com/NYPL/nypl-core/blob/master/vocabularies/csv/locations.csv#L2265
+    const hasOtfLocation = this.locations && this.locations[0] && this.locations[0].code === 'os'
+
+    // Having 910 $a === 'RLOTF' is also a solid indicator introduced in OTF
+    // record creation Oct 2021 https://github.com/NYPL/recap-hold-request-consumer/pull/33
+    const varfield910 = this.varField('910', 'a')
+    const hasOtf910 = Array.isArray(varfield910) && varfield910[0] && varfield910[0].value === 'RLOTF'
+
+    return hasOtfLocation || hasOtf910
+  }
+
+  getSuppressionRationale () {
+    if (this.suppressed) return 'suppressed'
+    if (this.deleted) return 'deleted'
+    if (this.isOtfRecord()) return 'is-otf'
+    return null
+  }
+
+  getSuppressionWithRationale () {
+    const rationale = this.getSuppressionRationale()
+    console.log('rationale: ', rationale)
```

Comment:
rm console.log? or is this supposed to be a logger.info or something?

Reply:
Fixed